homeassistant/components/netatmo.py

```diff
@@ -37,6 +38,7 @@
         vol.Required(CONF_PASSWORD): cv.string,
         vol.Required(CONF_SECRET_KEY): cv.string,
         vol.Required(CONF_USERNAME): cv.string,
+        vol.Required(CONF_DEVICES): cv.ensure_list,
```

Comment:
OK that's a bug with my components, I'll do a hotfix for this, but it should not be blocking